src/flask_restalchemy/api.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, blueprint=None, request_decorators=None):
     def init_app(self, blueprint):
         self._blueprint = blueprint
 
-    def add_model(self, model, url=None, serializer_class=None, view_name=None, request_decorators=None, methods=None):
+    def add_model(self, model, url=None, serializer_class=None, view_name=None, request_decorators=None, methods=None, query_callback=None):
```

Comment:
It would be nice to have a description in the docstring on how the query_callback should look like.

Reply:
Rename `query_callback` to `query_modifier`